lighthouse-core/audits/image-size-responsive.js

```diff
@@ -0,0 +1,198 @@
+/**
+ * @license Copyright 2020 Sebastian Kreft All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+/**
+ * @fileoverview Checks to see if the size of the visible images used on
+ *   the page are large enough with respect to the pixel ratio. The
+ *   audit will list all visible images that are too small.
+ */
+'use strict';
+
+const Audit = require('./audit.js');
+const URL = require('../lib/url-shim.js');
+const i18n = require('../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Title of a Lighthouse audit that provides detail on the size of visible images on the page. This descriptive title is shown to users when all images have correct sizes. */
+  title: 'Displays images with correct size',
+  /** Title of a Lighthouse audit that provides detail on the size of visible images on the page. This descriptive title is shown to users when not all images have correct sizes. */
+  failureTitle: 'Displays images with incorrect size',
+  /** Description of a Lighthouse audit that tells the user why they should maintain the correct aspect ratios for all images. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
+  description: 'Image natural dimensions should be proportional to the display size and the ' +
+    'pixel ratio. [Learn more](https://web.dev/image-size-responsive).',
```

Comment:
```suggestion
    'pixel ratio to maximize image clarity. [Learn more](https://web.dev/image-size-responsive).',
```

or something? right now it doesn't really feel as if it tells the user *why* to care :)